### PR TITLE
chore: remove stale entries from gitignore

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
chore

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
NA
**Summary**

remove stale entries from gitignore

**Does this PR introduce a breaking change?**

**Other information**
no

### CODE REVIEW

File: scripts/cleanupTest.js [hunk 1]
```diff
@@ -3,16 +3,16 @@ const rimraf = require('rimraf');
 const { join } = require('path');
 const collectTestFolders = require('./utils');
 
-const outputDirectories = ['bin', 'binary', 'dist', 'test', 'test-assets', 'test-plugin', 'test-loader', 'stats.json'];
+const outputDirectories = ['bin', 'binary', 'dist', 'lib', 'test', 'test-assets', 'test-plugin', 'test-loader', 'stats.json'];
```
Comment (anshumanv) [inline scripts/cleanupTest.js:6]: which test is creating lib?

Comment (snitin315) [reply scripts/cleanupTest.js:6]: `loader-error-test` but it's creating it at the root `/lib` but I think the script cleans only folders created inside the test folder `/test/**`?

Comment (anshumanv) [reply scripts/cleanupTest.js:6]: yep base dir is test, maybe create the output file in the test directory itself and name it `dist` or something

Comment (snitin315) [reply scripts/cleanupTest.js]: updated 👍🏻

**Review by anshumanv — APPROVED** [review]

**Review by gairTanm — APPROVED** [review]

**Review by evenstensberg — APPROVED** [review]

**Pull request merged**